src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2315,6 +2315,8 @@ public abstract partial class Enum : System.ValueType, System.IComparable, Syste
         [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the GetValues<TEnum> overload instead.")]
```

Comment:
```suggestion
        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the GetValues<TEnum> overload or GetValuesAsUnderlyingType method instead.")]
```

Reply:
(fix all variants of this message)